### DISCUSSION TITLE
How to set reserve price in listing NFT with "Sell to highest bidder" type

### DISCUSSION THREAD

**Discussion by lehongvo** [ProjectOpenSea/opensea-js#1242] [Q&A] [answered]
### Component

API

### Have you ensured that all of these are up to date?

- [X] opensea-js
- [X] Node (minimum v16)

### What version of opensea-js are you on?

6.1.11

### What function is the bug in?

createSellOrder

### Operating System

macOS (Apple Silicon)

### Describe the bug

<img width="744" alt="image" src="https://github.com/ProjectOpenSea/opensea-js/assets/71160314/edf97281-fd53-4da4-a567-00c992e3907f">
`orderData = {
		asset: {
			tokenAddress: tokenAddress,
			tokenId: tokenId,
			tokenStandard: constants.TOKEN_STANDARD
		},
		accountAddress: accountAddress,
		startAmount: startAmount,
		reserveAmount: 'amount', // please confirm this field, I don't see this field exists in params of openseaSDK
		paymentTokenAddress: paymentTokenAddress,
		expirationTime: expirationTime,
		englishAuction: isListingAuction,
	}
let order = await openSeaSdk.createSellOrder(orderData)	
`

**Comment by ryanio** [comment]
~~there is no reserve price functionality on opensea~~

**Reply by lehongvo** [reply]
@ryanio I see your UI, 
<img width="787" alt="image" src="https://github.com/ProjectOpenSea/opensea-js/assets/71160314/ffcc050f-ec99-4ed8-a80b-17c92f196c21">
How to input this param from openseaSdK

**Reply by ryanio** [reply]
oops sorry about that i missed that, let me look into it for you if it is possible through the API

**Reply by ryanio** [reply, marked as answer]
Hi @lehongvo i checked for you and we don't currently have this param as a feature in the api. Will keep it in mind as a request, thanks! We probably won't get to it for at least a couple months unfortunately we have many items on the backlog.
